tests/ftests/065-sudo-systemd_cgclassify-v1.py

```diff
@@ -93,8 +93,8 @@ def create_process_get_pid(config, CGNAME, SLICENAME='', ignore_systemd=False):
 
```

Comment:
I'm not 100% certain that I updated this test properly.  I did intentionally force errors and verify that the error paths behave mostly sanely, but it would be good to have another set of eyes look at this.